### DISCUSSION TITLE
Alt text - Needs support ASAP

### DISCUSSION THREAD

**Discussion by Nicholas-Arthur-Cook** [react-icons/react-icons#583] [General]
Hi react-icons team. I'm currently messing around with your icons on one of my webpages, and I noticed that there is no way to specify the alt text attribute for the images in the icons. This leads to a screen reader just reading "image" for each icon, with a screen reader user having no way to identify what the icon itself does. 

This is an **extremely important issue** because it means that every single site using these icons is not WCAG compliant, because all images are required to have alt text. 
The way that I was planning on using the alt text for the icon was to say what the button itself would do, or give it the null alt text attribute using alt="".  Please let me know if there is something wrong with how I am using the icons, or if there is a way I could contribute to the project with a PR myself.

Thanks!
Nick

**Comment by theDanielJLewis** [comment]
Still no support for this?
